README.md

```diff
@@ -1,2 +1,109 @@
-# bamboo-viability
-🎍 Findings from our EVE Bamboo I viability assessment
+# Bamboo I Viability Assessment
+
+- [Overview](#overview)
+- [Problems areas](#problem-areas)
+  - [Data access](#data-access)
+  - [Design components](#design-components)
+  - [Out-of-band decisions](#out-of-band-decisions)
+  - [Participant accounts](#participant-accounts)
+- [Tasks for CGA](#tasks-for-cga)
+- [Architecture diagram](#architecture-diagram)
+- [Prototype](#prototype)
+
+## Overview
+
+NewAperio has completed a viability assessment for the Bamboo I project (hereafter referred to as _Bamboo_) to ensure that the necessary technology requirements can be met. After a deep dive into the existing [Common Ground for Action (CGA)][cga] application and data model, and collaboration with the [Every Voice Engaged (EVE)][eve] technology team, we’ve uncovered a well-defined problem space and have come up with solutions that will allow the project to move forward.
+
+## Problem areas
+
+These are the problem areas that we identified, and are described in further detail later in the document:
+
+1. Data access
+2. Design components
+3. Out-of-band decisions
+4. Participant accounts
+
+### Data access
+
+#### The problem
+
+Data essential to the Bamboo experience is tied into the CGA environment and not exposed via a web API or other such mechanism. This data includes things like:
+
+- Issue guides
+- Options
+- Actions
+- Drawbacks
+
+#### The solution
+
+NewAperio will build a proxy server application that will connect directly to CGA’s database instance via a read-only connection and expose the required data over a well-defined GraphQL web API.
+
+We discussed other options with the CGA team that would involve them making non-neglible changes to the exising legacy codebase:
+
+- Experimenting with [Lift’s REST helpers][lift-rest]
+- Building a bespoke [JSON:API] service
+
+After evaluating the tradeoffs, both teams decided that the solution we chose would be best--it ensures we have access to the data Bamboo needs and requires no modification of the legacy app.
+
+### Design components
+
+#### The problem
+
+Some of the content included in the [prototype designs] are not available in the CGA application. This includes:
+
+- Issue guide cover images
+- Issue guide categories
+
+#### The solution
+
+After consulting with our senior designer, we feel like we can build Bamboo without this content and not unduly compromise the user experience.
+
+This content, though not essential, would be nice to have and we’ve identified a couple of potential pathways for getting access to it:
+
+- The CGA team could add fields to the legacy app to allow issue guide authors to specify an image and category
+- The [EVE website][eve], a WordPress app, has the content we need and there are plugins for WordPress that can provide a drop-in web API. The difficult part of this approach, however, is linking up the WordPress data with the correct CGA issue guides.
```

Comment:
I think WP has custom fields, they could add a custom field with the database ID?

Reply:
@lleger Yep, that would be a good way to handle it.